### ISSUE TITLE
chore: Audit components, cleanup

### ISSUE THREAD

**Issue by kevee** [COVID19Tracking/website#1318]
Our list of components has grown pretty long, and some are no longer being used. **AFTER** #1317 is merged in, let's:

- [x] Audit components and remove ones that are no longer used
- [x] Restructure components into deeper-nested directories
- [x] Move around any scss files not in the components directory
- [x] Change all our unnamed `default components` to named default components
- [x] Move visualization guide to it's own sub-component folder, rename to kebab case

Also:

- [x] Audit snippets in Contentful and unpublish ones that are not being used

**Comment by schwartzadev** [comment]
Is this still an open item, considering e73b7e20d5d9527f1006ebc66951abc35acc1cb5?

If it is, @kevee let's touch base and I can take this on.

**Comment by schwartzadev** [comment]
Re: relational imports --> I put together a webpack shim that tells depcruise how to handle the our aliases in the `depcruise-relational-imports-1318` branch. Once #1317 is merged into master, we can merge master into this branch and run depcruise there with:

```
depcruise --max-depth 2 --exclude "^(node_modules|src/stories|src/__tests__|src/images)" --webpack-config ./webpack/webpack.config.js --output-type dot src | dot -T svg > dependencygraph.svg
```

**Comment by kevee** [comment]
Thanks! Wow that dep chart is scary...
